packages/flutter/lib/src/material/button_style_button.dart

```diff
@@ -155,14 +157,20 @@ abstract class ButtonStyleButton extends StatefulWidget {
   /// Defaults to true.
   final bool? isSemanticButton;
 
+  /// {@macro flutter.material.ButtonStyleButton.iconAlignment}
+  final IconAlignment iconAlignment;
+
+  /// Text that describes the action that will occur when the button is pressed.
+  ///
+  /// This text is displayed when the user long-presses on the button and is
```

Comment:
Just feel the original explanation is not super clear because the text will also get displayed when the button is on hovered on desktop platforms.

Reply:
Good point, i made some improvements. Also mentioned what happens if this parameter is `null`.